pkgs/by-name/rp/rpm-sequoia/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  nettle,
+  nix-update-script,
+  rustPlatform,
+  pkg-config,
+  runCommand,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rpm-sequoia";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "rpm-software-management";
+    repo = "rpm-sequoia";
+    rev = "v${version}";
+    hash = "sha256-AZCsboUv4muKOw5El2Hw5O1cvAgD3JhBppacrQCJT2k=";
+  };
+
+  cargoHash = "sha256-0yO1+OAkXje/ir8i8URVhIcW8gwXleYx+XL1U4bjtXk=";
+
+  patches = [
+    ./objdump.patch
+    (substituteAll {
```

Comment:
I would prefer `replaceVars` here.

Reply:
I dropped the patch
See:
https://github.com/NixOS/nixpkgs/pull/354645#discussion_r1837080260